src/CoreWCF.Http/src/CoreWCF/Channels/HttpRequestContext.cs

```diff
@@ -348,6 +348,31 @@ public AspNetCoreHttpContext(IHttpTransportFactorySettings settings, HttpContext
 
             public override string HttpMethod => _aspNetContext.Request.Method;
 
+            public override bool KeepAliveEnabled
+            {
+                get
+                {
+                    string connectionHeader = HttpRequestHeader.Connection.ToString();
+
```

Comment:
I think using nameof(HttpRequestHeader.Connection) would achieve the same thing but no incur the cost of calling Enum.ToString() which historically has been really expensive although I think they did improve it. If you use nameof, I think you can make this into a private const string. You're calculating the string value on every request. If you use nameof though, it will just be turned into a const string by the compiler anyway so you could keep it inline.

Reply:
Can you add a TODO comment in here as .NET 6 expanded IHeaderDictionary to have properties for each of the common headers. Once we drop support for earlier versions, we can do _aspNetContext.Request.Headers.Connection instead.